subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenPomResolveIntegrationTest.groovy

```diff
@@ -30,13 +30,14 @@ class MavenPomResolveIntegrationTest extends AbstractHttpDependencyResolutionTes
     <version>1.2</version>
     <distributionManagement>
         <relocation>
```

Comment:
We should have two different test cases here:

1. Usage of just `groupId`.
2. Usage of `groupId` and `artifactId` together.

Based on the Maven specification I could not identify if the use of just `artifactId` is a valid possibility. Maybe you can try this out locally by running Maven and see how it behaves.